buildSrc/src/main/java/Config.kt

```diff
@@ -82,6 +83,8 @@ object Config {
         val retrofit2Gson = "$retrofit2Group:converter-gson:$retrofit2Version"
 
         val coroutinesCore = "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3"
+
+        val fragment = "androidx.fragment:fragment-ktx:1.3.4"
```

Comment:
the `-ktx` is written in Kotlin so it'd be fine if the whole module is written in Kotlin, but the base of this library is written in Java and could be used in plain Java as well https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:fragment/fragment/src/main/java/androidx/fragment/app/

to be honest, I also prefer to be written in Kotlin, but we've always to think about a final user that has a Java app and don't want to bring the Kotlin stblib in as a transitive dependency.

@bruno-garcia and @philipphofmann opinions? I'd go with Kotlin IMO